ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiWorldStateProvider.java

```diff
@@ -337,7 +335,7 @@ public TrieLogManager getTrieLogManager() {
 
   @Override
   public void resetArchiveStateTo(final BlockHeader blockHeader) {
-    persistedState.resetWorldStateTo(blockHeader);
+    persistedState.setArchiveStateUnSafe(blockHeader);
```

Comment:
This seems out of place, merge/rebase issue?

Reply:
rebase issue.  It was next to some `fromPlugin` removals.